src/main/java/com/griddynamics/cloud/learning/web/dto/ArticleRequestFilter.java

```diff
@@ -0,0 +1,11 @@
+package com.griddynamics.cloud.learning.web.dto;
+
+import lombok.Data;
+
+@Data
+public class ArticleRequestFilter {
```

Comment:
-> ArticleFilterRequest

You know, it'd be better to have PaginatedRequestParams as a field here

Reply:
rename -> ArticleFilterRequest